File: Python/7py 1/add_binary.py
"""
|--------------------------------------------------------------------------
| Binary Addition
|--------------------------------------------------------------------------
|
"""

def add_binary(a,b):
   return bin(a+b)[2:]


def add_binary(a,b):
   return '{0:b}'.format(a + b)


def add_binary(a, b):
   return format(a + b, 'b')


def add_binary(a,b):
   return f"{a + b:b}"


print(add_binary(1,1),"10")
print(add_binary(0,1),"1")
print(add_binary(1,0),"1")
print(add_binary(2,2),"100")
print(add_binary(51,12),"111111")

File: PHP/7php 1/add_binary.php
<?php
/*
|--------------------------------------------------------------------------
| Binary Addition
|--------------------------------------------------------------------------
|
*/

function add_binary($a, $b) {
   return decbin($a + $b);
}


function add_binary($a, $b) {
   $out = "";
   $sum = $a + $b;
   if (!$sum) return "0";
   for (; $sum>0; $sum=$sum>>1) {
      $out = (($sum%2)?"1":"0") . $out;
   }
   return $out;
}


function add_binary($a, $b) {
   $binary = $a + $b;
   return decbin($binary);
}


print_r(add_binary(1, 1) . " 10
");
print_r(add_binary(5, 9) . " 1110
");